aptos-move/framework/aptos-framework/doc/chain_status.md

```diff
@@ -109,7 +109,7 @@ Marks that genesis has finished.
 
 <a name="0x1_chain_status_is_genesis"></a>
 
-## Function `is_genesis`
+## `#[view]`<br>Function `is_genesis`
 
```

Comment:
I would have expected the `#[view]` in front of the function declaration but not as part of the title of the markdown section. The current treatment may be rather problematic if the attributes are larger. Also, the attribute doesn't contribute meaningful to the title or TOC.